### ISSUE TITLE
Brave randomly turning off shield for random websites

### ISSUE THREAD

**Issue by DgitalTrex** [brave/brave-browser#23113]
## Description 
Brave keeps turning off brave shields randomly on all sites. I have to keep turning it on. even then, when I come back to the same site, it's off again. Checking "global settings" shows absolutely nothing on why this is happening

## Steps to Reproduce

1. Open a website
2. Find out that shield is off
3. Turn shield on
4. Close tab
5. Open tab again same website

## Actual result:
brave shield to keep on on all websites

## Expected result:
brave shield is frustratingly randomly off

## Reproduces how often: 
intermittent since the behavior is random

## Brave version (brave://version info)
1.39.111 Chromium: 102.0.5005.61 (Official Build) (64-bit) 

## Version/Channel Information:
I don't know i use the official channel only

## Other Additional Information:
N/A

## Miscellaneous Information:
N/A

**Comment by manikandancode** [comment]
Yes. After 1.39.111 update. plz check

**Comment by DgitalTrex** [comment]
> Yes. After 1.39.111 update. plz check

You have the same issue?

**Comment by manikandancode** [comment]
Yes. The shields are turned off on all site. Tried changing standard and Aggressive in global but no effect. Need to enable the shields individually on all domains. Only on my Windows 7 system. But working well in my Windows 10.

> Brave	1.39.111 Chromium: 102.0.5005.61 (Official Build) (64-bit) 
> Revision	0e59bcc00cc4985ce39ad31c150065f159d95ad3-refs/branch-heads/5005@{#819}
> OS	Windows 7 Service Pack 1 (Build 7601)
> JavaScript	V8 10.2.154.4
> User Agent	Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.5005.61 Safari/537.36
> Command Line	"C:\Program Files (x86)\BraveSoftware\Brave-Browser\Application\brave.exe" --disable-client-side-phishing-detection --disable-domain-reliability --enable-dom-distiller --no-pings --component-updater=url-source=https://go-updater.brave.com/extensions --origin-trial-public-key=bYUKPJoPnCxeNvu72j4EmPuK7tr1PAC7SHh8ld9Mw3E=,fMS4mpO6buLQ/QMd+zJmxzty/VQ6B1EUZqoCU04zoRU= --sync-url=https://sync-v2.brave.com/v2 --lso-url=https://no-thanks.invalid --variations-server-url=https://variations.brave.com/seed --variations-insecure-server-url=https://variations.brave.com/seed --flag-switches-begin --flag-switches-end
> Executable Path	C:\Program Files (x86)\BraveSoftware\Brave-Browser\Application\brave.exe
> Profile Path	C:\Users\Server\AppData\Local\BraveSoftware\Brave-Browser\User Data\Default
> Variations	AdRewardsStudy:NextPaymentDay
> AdServingStudy:DefaultAdNotificationsPerHour=10/MaximumAdNotificationsPerDay=100/MaximumInlineContentAdsPerHour=12/MaximumInlineContentAdsPerDay=60
> BraveAds.UserActivityStudy:Triggers=EMPTY/Threshold=0.0/IdleTimeThreshold=5
> BraveDebounceStudy:Enabled
> ConversionsStudy:NewDefaultPattern
> Default1pBlockingStudy:Disabled
> EphemeralStorageStudy:Enabled
> ExtendedVariationsSafeMode5:SignalAndWriteViaFileUtil5
> FirstPartyEphemeralDomainBlockStudy:Enabled
> ForceDisableBlinkFeatures:Enabled
> NativeCosmeticFilteringStudy:Enabled
> NavigationThreadingOptimizationsCompat:Disabled
> PartitionBlinkMemoryCacheStudy:Enabled
> PartitionConnectionsByNetworkIsolationKeyStudy:Enabled
> PermissionLifetimeReleaseStudy:Enabled
> RequestAdsEnabledApiStudy:Enabled
> SpeedreaderReleaseStudy:Enabled
>

**Comment by Werve** [comment]
I confirm I've the same issue, Shields disabled on default but works (persist setting on) if manually re enabled on each site.
Probably is the cause of #1897 

```
Brave | 1.39.111 Chromium: 102.0.5005.61 (Build ufficiale) (a 64 bit)
Revisione | 0e59bcc00cc4985ce39ad31c150065f159d95ad3-refs/branch-heads/5005@{#819}
Sistema operativo | Windows 10 Version 21H2 (Build 19044.1706)
JavaScript | V8 10.2.154.4
User-agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.5005.61 Safari/537.36
Riga di comando | "C:\Program Files (x86)\BraveSoftware\Brave-Browser\Application\brave.exe" --disable-client-side-phishing-detection --disable-domain-reliability --enable-dom-distiller --no-pings --component-updater=url-source=https://go-updater.brave.com/extensions --origin-trial-public-key=bYUKPJoPnCxeNvu72j4EmPuK7tr1PAC7SHh8ld9Mw3E=,fMS4mpO6buLQ/QMd+zJmxzty/VQ6B1EUZqoCU04zoRU= --sync-url=https://sync-v2.brave.com/v2 --lso-url=https://no-thanks.invalid --variations-server-url=https://variations.brave.com/seed --variations-insecure-server-url=https://variations.brave.com/seed --flag-switches-begin --ignore-gpu-blocklist --flag-switches-end
Percorso eseguibile | C:\Program Files (x86)\BraveSoftware\Brave-Browser\Application\brave.exe
Percorso profilo | C:\Users\Antonio\AppData\Local\BraveSoftware\Brave-Browser\User Data\Default
Varianti | AdRewardsStudy:NextPaymentDayAdServingStudy:DefaultAdNotificationsPerHour=10/MaximumAdNotificationsPerDay=100/MaximumInlineContentAdsPerHour=12/MaximumInlineContentAdsPerDay=60BraveAds.UserActivityStudy:Triggers=EMPTY/Threshold=0.0/IdleTimeThreshold=5BraveDebounceStudy:EnabledConversionsStudy:NewDefaultPatternDefault1pBlockingStudy:DisabledEphemeralStorageStudy:EnabledExtendedVariationsSafeMode5:SignalAndWriteViaFileUtil5FirstPartyEphemeralDomainBlockStudy:EnabledForceDisableBlinkFeatures:EnabledNativeCosmeticFilteringStudy:EnabledNavigationThreadingOptimizationsCompat:DisabledPartitionBlinkMemoryCacheStudy:EnabledPartitionConnectionsByNetworkIsolationKeyStudy:EnabledPermissionLifetimeReleaseStudy:EnabledRequestAdsEnabledApiStudy:EnabledSpeedreaderReleaseStudy:Enabled
```

**Comment by manikandancode** [comment]
Annoying. switching to Vivaldi.
tested with : 1.39.122 Chromium: 102.0.5005.115 (Official Build) (64-bit) but still same.
Any help?

**Comment by leotm** [comment]
same on `1.39.122 Chromium: 102.0.5005.115 (Official Build) (64-bit)`

this fixed it for me: https://community.brave.com/t/problem-brave-shields-they-are-disabled-by-default/384188/14

![Capture](https://user-images.githubusercontent.com/1881059/174803640-083b5d8c-5311-42b5-8afd-471274bdee6b.PNG)

i had 2k sites whitelisted, so only if you're prepared to manually unblock each again

**Comment by rebron** [comment]
Closing as dupe of https://github.com/brave/brave-browser/issues/23214 and fixed in 1.41.x. Please reopen if updating to Brave 1.41.96 doesn't resolve this issue.
